.github/workflows/build_deploy_docs_release.yml

```diff
@@ -76,3 +76,4 @@ jobs:
         uses: ad-m/github-push-action@v0.6.0
         with:
           branch: gh-pages
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Unexpected that this is not the default?

Reply:
During release of 0.3.0 it appeared that this line has to be added, otherwise it does not work.